### ISSUE TITLE
Bug: program panics when using filter.

### ISSUE THREAD

**Issue by sidkhuntia** [JammUtkarsh/pms#8]
[Screencast from 07-11-23 12:15:57 AM IST.webm](https://github.com/JammUtkarsh/pms/assets/77291662/da32d483-b7eb-4502-99ce-fb32c6dacd19)
#### Steps to re-produce error
1. `pms`
2. `/` (filter option)
3.  Type in random letter 
4. press backspace

**Comment by jammutkarsh** [comment]
Thanks, I think this was introduced when we set backspace as a way to delete the project.
Commit: [54ab9da](https://github.com/JammUtkarsh/pms/commit/54ab9daa10e2e7e04094fe0264a20531e2706947)

Do you have any suggestion for how we can delete a project?

_Thanks for reporting._

**Comment by sidkhuntia** [comment]
Can we use a combination of keystrokes like `Shift + Backspace` or something like that? The first time I used it I pressed `Backspace` to exit the TUI but then it deleted the project record. In my opinion having  `Backspace` is not very intuitive

**Comment by jammutkarsh** [comment]
Yes, we could use a combination of keystrokes but this requires us to store the _last key press_ in buffer. I haven’t tested or used it yet. But if when we implement this, I thought of adding something similar to vim key binding.

> In my opinion having `Backspace` is not very intuitive

Right, until we try and test the `key buffer` we can set delete to `#`. This is not directly accessible and requires pressing two keys `shift` + `3`.

**Comment by sidkhuntia** [comment]
Sounds good to me.

**Comment by jammutkarsh** [comment]
Okay, then for the time being we can set the delete to `#` sign.

We can implement the multi-stroke keybinds later.

**Comment by jammutkarsh** [comment]
Conversation about [multi-key bind like vim](https://discord.com/channels/1032368467246075914/1138100316370387016/1138192172127092736)

Discord Invite Link: https://charm.sh/chat/

**Comment by sidkhuntia** [comment]
I joined the sever, but i guess the dicussion is in a private channel which i cant access

**Comment by jammutkarsh** [comment]
In _nerd corner/help_ search for "Does Bubbletea offer 'Multiple Key Press Keybinding' option like vim?"
